project-minimization-plugin/src/main/kotlin/org/plan/research/minimization/plugin/benchmark/BenchmarkResultAdapter.kt

```diff
@@ -0,0 +1,11 @@
+package org.plan.research.minimization.plugin.benchmark
+
+import org.plan.research.minimization.plugin.errors.MinimizationError
+
+import com.intellij.openapi.project.Project
+
+interface BenchmarkResultAdapter {
```

Comment:
Subscriber?

Reply:
Named with look at `ProcessResultAdapter` from Platform, so don't know what is better

Reply:
Subscriber :) Adapter is more about adapting something that is not ours, here we just subscribe to some events (our own events)